mopidy_listenbrainz/backend.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar, NewType
+
+import pykka
+
+from mopidy import backend
+
+try:
+    from mopidy.types import UriScheme
+except ModuleNotFoundError:
+    UriScheme = NewType("UriScheme", str)
+
+from . import playlists
+
+if TYPE_CHECKING:
+    from mopidy.audio import AudioProxy
+    from mopidy.ext import Config
+
+
+class ListenbrainzBackend(pykka.ThreadingActor, backend.Backend):
+    uri_schemes: ClassVar[list[UriScheme]] = [UriScheme("listenbrainz")]
+
+    def __init__(
+        self,
+        config: Config,  # noqa: ARG002
+        audio: AudioProxy,  # noqa: ARG002
+    ) -> None:
+        super().__init__()
+        self.playlists = playlists.ListenbrainzPlaylistsProvider(self)
```

Comment:
```suggestion
        self.playlists = ListenbrainzPlaylistsProvider(self)
```
nitpick but i would prefer to import like this here

Reply:
would also need to change line 14